### PR TITLE
Add  more software

### PR DESCRIPTION
**Pull request by hipudding**

### CODE REVIEW

File: _posts/2023-08-21-KAE.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+permalink: /:title/
+category: libraries
+
+meta:
+  keywords: "KAE"
+
+project:
+  title: "KAE"
+  type: "C++"
+  url: "https://github.com/kunpengcompute/KAEzip"
```
Comment (martin-g) [inline _posts/2023-08-21-KAE.md]:
```suggestion
  url: "https://github.com/kunpengcompute/KAE"
```

File: _posts/2023-08-21-KAE.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  project_official_url: "https://github.com/kunpengcompute/KAEzip"
```
Comment (martin-g) [inline _posts/2023-08-21-KAE.md]:
```suggestion
  project_official_url: "https://github.com/kunpengcompute/KAE"
```

File: _posts/2023-08-21-KAE.md (hunk 1 above)
Comment (hipudding) [reply _posts/2023-08-21-KAE.md]: My mistake, fixed in second commit.

File: _posts/2023-08-21-KAE.md (hunk 2 above)
Comment (hipudding) [reply _posts/2023-08-21-KAE.md]: My mistake, fixed in second commit.

**Review by ZhengZhenyu — APPROVED** [review]

**Pull request merged**